cookbooks/aws-parallelcluster-platform/resources/intel_hpc/intel_hpc.rb

```diff
@@ -22,5 +22,66 @@
 end
 
 action :configure do
-  # do nothing
+  return unless intel_hpc_supported? && (node['cluster']['install_intel_base_toolkit'] == "true" || node['cluster']['install_intel_hpc_toolkit'] == "true" || node['cluster']['install_intel_python'] == "true")
+
+  base_toolkit_name = "l_BaseKit_p_2023.2.0.49397_offline.sh"
+  base_toolkit_url = "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/#{base_toolkit_name}"
+  hpc_toolkit_name = "l_HPCKit_p_2023.2.0.49440_offline.sh"
+  hpc_toolkit_url = "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/0722521a-34b5-4c41-af3f-d5d14e88248d/#{hpc_toolkit_name}"
+  intel_python_name = "l_pythoni39_oneapi_p_2023.2.0.49422_offline.sh"
+  intel_python_url = "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/03aae3a8-623a-47cf-9655-5dd8fcf86430/#{intel_python_name}"
+  # Below are critical security updates not included in the tookits:
+  cpp_compiler_name = "l_dpcpp-cpp-compiler_p_2023.2.1.8_offline.sh"
+  cpp_compiler_url = "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/ebf5d9aa-17a7-46a4-b5df-ace004227c0e/#{cpp_compiler_name}"
+  fortran_compiler_name = "l_fortran-compiler_p_2023.2.1.8_offline.sh"
+  fortran_compiler_url = "https://registrationcenter-download.intel.com/akdlm/IRC_NAS/0d65c8d4-f245-4756-80c4-6712b43cf835/#{fortran_compiler_name}"
+
+  intel_offline_installer_dir = '/opt/intel/offlineInstaller'
+
+  directory intel_offline_installer_dir do
+    recursive true
+  end
+
+  if node['cluster']['install_intel_base_toolkit'] == "true"
+    install_intel_software "Install Intel Software" do
```

Comment:
Currently every resource `install_intel_software` has the same title "Install Intel Software", but every one of them installs different things. I suggest to reflect these differences in the title.

Reply:
Done